genai-on-vertex-ai/gemini/model_upgrades/instruction-following/vertex_colab/instruction_following_eval.ipynb

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
<p>Please clear all the cell outputs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>